src/plots/plot.ts

```diff
@@ -100,9 +100,11 @@ export class Plot extends Component {
    * Adds a Dataset to the Plot.
    *
    * @param {Dataset} dataset
+   * @param {boolean} callUpdateHandler Whether the typical on-dataset-update handler should be called.
+   *   If not provided, defaults to true.
    * @returns {Plot} The calling Plot.
    */
-  public addDataset(dataset: Dataset) {
+  public addDataset(dataset: Dataset, callUpdateHandler: boolean = true) {
```

Comment:
The explicit typing of  `boolean` is unnecessary here, since the type can be inferred from the fact that the value is set to `true`.